Utilities/sysinfo.cpp

```diff
@@ -38,6 +38,13 @@ bool utils::has_avx2()
 	return g_value;
 }
 
+bool utils::has_avx512()
```

Comment:
Don't move function code like this

Reply:
Am I supposed to fix this? Or is it ok this way?

Reply:
Maybe you can show me, how to do it correctly? 
I am just not familiar with cpp code rules. I should say I'm a newbie.

Reply:
Moving a function makes reading through git history annoying. Your commit will appear as the one which added this function, and it's cubersome to find further history details about the function. Keep the function placed where it was, so that only the name appears to have been changed.

Reply:
Can you help me fix this, or I should just create another PR without moving the function?

Reply:
Move the function back to where it was and then make it into another commit. You'll have two commits for one PR, but they can be combined into one when merging.

If you want to combine the commits yourself and are using the command line git, then run `git rebase -i HEAD~2`, mark the latter commit as fixup and then force push the changes back into your own repository. If this sounds too complicated then don't bother with it, just add the change as another commit and push it normally.

If you want more help, use the developement channel on our discord server.

Reply:
Well, changed as you said. 
Again, I hope that I didn't break anything.